### DISCUSSION TITLE
Deploy only when a release tag is created on the github repo

### DISCUSSION THREAD

**Discussion by vacom** [vercel/vercel#4121] [General] [answered]
It is possible to only trigger a deployment to production only when a release tag is created?

**Comment by styfle** [comment, marked as answer]
No, this is not currently possible.

Production deployments are created from the git repo's default branch ([see docs](https://vercel.com/docs/v2/platform/deployments/#production)).

Can you explain a little more about your current workflow?

**Reply by vacom** [reply]
Hey @styfle,

Currently, I have the workflow adjusted to Vercel, based on environments with different URLs, in this case, everything that is placed in the Master branch is published for production.

this is not a problem I just wanted to know if it was possible but I can adapt it with the current workflow.
